src/component/Form.js

```diff
@@ -0,0 +1,13 @@
+import './styles/form.css';
+
+export default function Form() {
+  return (
+    <form>
+      <span>Add new book</span>
+      <div>
+        <input placeholder="book title" />
+        <button type="button">ADD BOOK</button>
+      </div>
+    </form>
```

Comment:
- [ ] You have done extremely well so far, :clap: However, you are also required to add an input field for the author and its placeholder, just like what you did for the book tile. Please resolve that. :slightly_smiling_face: